lib/galaxy/webapps/galaxy/api/workflows.py

```diff
@@ -854,10 +854,22 @@ def __get_stored_workflow(self, trans, workflow_id, **kwd):
         "is:published",
         "Include only published workflows in the final result. Be sure the query parameter `show_published` is set to `true` if to include all published workflows and not just the requesting user's.",
     ),
+    IndexQueryTag(
+        "is:importable",
+        "Include only importable workflows in the final result.",
+    ),
+    IndexQueryTag(
+        "is:deleted",
+        "Include only deleted workflows in the final result.",
+    ),
     IndexQueryTag(
         "is:share_with_me",
```

Comment:
```suggestion
        "is:shared_with_me",
```
Not related directly to the changes here but maybe this typo could also be fixed here...